src/content/energy-consumption/index.md

```diff
@@ -4,65 +4,66 @@ description: The basic information you need to understand Ethereum's energy cons
 
 ## Ethereum's carbon debt {#carbon-debt}
 
-Ethereum's current energy expenditure is very low, but this has not always been the case. Ethereum switched on its proof-of-stake consensus mechanism in Q3 2022. However, Ethereum used a proof-of-work mechanism from 2014-2022, which had a much greater environmental cost.
+Ethereum's energy expenditure is very low, but this has not always been the case. Ethereum originally used proof-of-work which had a much greater environmental cost than the current proof-of-stake mechanism.
 
-Since its inception, Ethereum aimed to implement a proof-of-stake consensus mechanism, but doing so without sacrificing security and decentralization took years of focused research and development. Therefore, a proof-of-work mechanism was used to get the network started. Proof-of-work consensus requires miners to use their computing hardware to solve a puzzle, expending energy in the process. The solution to the puzzle proves that energy has been expended by the miner, demonstrating that they invested real-world value for the right to add to the blockchain. Ethereum's total energy consumption peaked during the apex of the crypto bull market in February 2022 at just under 94 TWh/yr. In the summer before the switch to proof-of-stake, the energy consumption was closer to 60 TWh/yr, comparable to that of Uzbekistan, with a carbon emission equivalent to that of Azerbaijan (33 MT/yr).
-
-[CCRI](https://carbon-ratings.com) has examined the impact of Ethereum’s merge from proof-of-work to proof-of-stake; the results have underlined the significant impact of changing the consensus protocol: The annualized electricity consumption was reduced from 22,900,320 MWh to 2,601 MWh and thus by more than **99.988 %**. Likewise, Ethereum’s carbon footprint was decreased by approximately **99.992 %** (from 11,016,000 to 870 tonnes CO2e). Depicted metaphorically, this corresponds to a reduction in emissions from the height of the Eiffel Tower to a small plastic toy figure, as shown in the figure below. 
+From the very beginning, Ethereum aimed to implement a proof-of-stake based consensus mechanism, but doing so without sacrificing security and decentralization took years of focused research and development. Therefore, a proof-of-work mechanism was used to get the network started. Proof-of-work requires miners to use their computing hardware to calculate a value, expending energy in the process. Ethereum's total energy consumption peaked during the apex of the crypto bull market in February 2022 at just under 94 TWh/yr. Just before the switch to proof-of-stake, the energy consumption was closer to [78 TWh/yr](https://digiconomist.net/ethereum-energy-consumption), comparable to that of Uzbekistan, with a carbon emission equivalent to that of Azerbaijan (33 MT/yr). The switch to proof-of-stake solved this problem.
```

Comment:
> The switch to proof-of-stake solved this problem.

Partially. It halts further progression of the problem, but does not undo the impact from the first seven years. Should we address this?

Reply:
Perhaps we tie the above reality to the ReFi stuff below, where actions are being taken beyond just the switch to PoS to further mitigate the environmental impact that has already been sustained.

Reply:
```suggestion
From the very beginning, Ethereum aimed to implement a proof-of-stake based consensus mechanism, but doing so without sacrificing security and decentralization took years of focused research and development. Therefore, a proof-of-work mechanism was used to get the network started. Proof-of-work requires miners to use their computing hardware to calculate a value, expending energy in the process. Ethereum's total energy consumption peaked during the apex of the crypto bull market in February 2022 at just under 94 TWh/yr. Just before the switch to proof-of-stake, the energy consumption was closer to [78 TWh/yr](https://digiconomist.net/ethereum-energy-consumption), comparable to that of Uzbekistan, with a carbon emission equivalent to that of Azerbaijan (33 MT/yr).
```